Task3/Task3_Atanas_Kisyov/db/README.md

```diff
@@ -0,0 +1,14 @@
+#Execute SQL using psql
```

Comment:
Please update steps. Again. You need to think about a user. User does not know anything about your program.....